docs/crio.8.md

```diff
@@ -134,8 +134,24 @@ it later with **--config**. Global options will modify the output.
 **--default**
   Output the default configuration (without taking into account any configuration options).
 
+## FILES
+
+**crio.conf** (`/etc/crio/crio.conf`)
+
+	crio.conf is the cri-o configuration file for all of the available command-line options for the crio(8) program, but in a TOML format that can be more easily modified and versioned.
```

Comment:
Nit: can we get backticks around the filenames and paths (e.g. `crio.conf` here)?

Reply:
I had them but go-md2man does not translate them so they look lousy in man page output.